src/compas_xr/assemblies/assemblies.py

```diff
@@ -0,0 +1,94 @@
+
+import os
+from compas.data import json_dumps,json_loads
+from compas_xr.storage import Storage
+from compas_xr.realtime_database import RealtimeDatabase
+
+class AssemblyAssistant(object):
+
+    def __init__(self, config_path):
+
+        if os.path.exists(config_path):
+            self.storage = Storage(config_path)
+            self.database = RealtimeDatabase(config_path)
+
+        # Raise Exception that it could not be founda at that path.
+        else:
+            raise Exception("Could not create Storage or Database with path {}!".format(config_path))
+  
+    #Functions for uploading Assemblies to the Database and Storage    
+    def upload_assembly_all_to_database(self, assembly, parentname): 
+
+        #Get data from the assembly
+        data = assembly.__data__
+
+        # Upload data using Database Class
+        self.database.upload_data_all(data, parentname)
+
+    #TODO: Integrated in toggle for removing joint keys.
+    def upload_assembly_to_database(self, assembly, parentname, parentparamater, parameters, joints):
+        
+        #Get data from the assembly
+        data = assembly.__data__
+
+        #Create Data Structure form Assembly
+        paramaters_nested = {}
+        
+        for param in parameters:
+            values = data[parentparamater][param]
+            parameters_dict = {param: values}
+            paramaters_nested.update(parameters_dict)
+
+        if (joints == False):
```

Comment:
The pythonic way to test for false-y is:
```suggestion
        if not joints:
```
(also, the extra parens are not needed)